src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -2769,7 +2787,9 @@ def set_up_network_profile(self, mc: ManagedCluster) -> ManagedCluster:
             network_profile.monitoring = self.models.NetworkMonitoring(  # pylint: disable=no-member
                 enabled=network_observability
             )
+        print("testonly2 !!!!!", network_profile.load_balancer_profile)
 
+        mc.network_profile = network_profile
```

Comment:
may not need this, this would be done in construct_mc_profile_preview/construct_mc_profile_default (from azure-cli).

Reply:
removed